updates to ega

thosgood opened the pull request:
there have been a tonne of changes to the structure of the ega project, so submitting this PR to check if things still work :-)

jakebian commented:
Thanks! Will check this but looks like there's merge conflicts - probably there was a force push to master branch of the original repo. Instead of rebasing this branch on top it might be drastically easier instead for me to check out your branch and force push to master of this repo 💀

Also remembering I meant to set a bot to automatically keep this fork up to date with the upstream, unfortunately never got to it, will try to find time to do.

thosgood committed:
started ega2-4.4

thosgood committed:
pause before this massive proposition

thosgood committed:
Merge pull request #166 from solov-t/master

First draft of subsection IV.16.6

solov-t committed:
Merge pull request #8 from ryankeleti/master

Merge from master

thosgood committed:
finished ega-2.4.4

thosgood committed:
Merge pull request #167 from ryankeleti/ega2-4

finished ega-2.4.4

solov-t committed:
Added \shDer and \shAut commands to the web version

thosgood committed:
Merge pull request #168 from solov-t/master

Created shDer and shAut command for the web version

solov-t committed:
Merge pull request #9 from ryankeleti/master

Merge from main

solov-t committed:
FIrst additions to IV.16.7

solov-t committed:
Zero'th draft of IV.16.7

Also fixed \isoto's

solov-t committed:
Minor typos

thosgood committed:
Merge pull request #169 from solov-t/master

First draft of IV.16.7

solov-t committed:
Start of section IV.16.8

thosgood committed:
starting again

thosgood committed:
starting a proof

thosgood committed:
updated readme for when this is finished

thosgood committed:
another page

thosgood committed:
typo

solov-t committed:
More subsubsections

thosgood committed:
another page or so

thosgood committed:
\setminus -> \setmin

solov-t committed:
Zeroth draft of subsection IV.16.8

thosgood committed:
this is a looong proof

solov-t committed:
Typos

ryankeleti committed:
Merge pull request #170 from solov-t/master

Subsection IV.16.8

solov-t committed:
Start of subsubsection IV.16.9

thosgood committed:
finished 2-4.5

thosgood committed:
almost finished 2-4.6

thosgood committed:
finished 2-4.6; finished ega2-4

thosgood committed:
Merge pull request #171 from ryankeleti/ega2-4

finished ega2-4

thosgood committed:
next goal

ryankeleti committed:
Clean up README a bit and add matrix.to link

Remove periods after PDF links.
Link to `matrix.to` instead of `riot.im` for matrix room.

solov-t committed:
more subsubsections

solov-t committed:
more subsubsections

solov-t committed:
Finished IV.16.9 and started IV.16.10

solov-t committed:
finished section 16.10

solov-t committed:
Just one more subsection, c'mon

solov-t committed:
The first draft of section IV.16

this is it

solov-t committed:
This felt good

thosgood committed:
Merge pull request #172 from solov-t/master

The first draft of section IV.16

ryankeleti committed:
Merge branch 'master' of https://github.com/ryankeleti/ega into ega4-20

ryankeleti committed:
Fix typos and phrasing

thosgood committed:
back on it

thosgood committed:
Merge pull request #173 from ryankeleti/ega2-2

small ega2-2 update; ega4-20

thosgood committed:
two things

thosgood committed:
tiny bit more

thosgood committed:
finished II.2.3

thosgood committed:
Merge pull request #174 from ryankeleti/ega2-2

finished II.2.3

thosgood committed:
fixed error

thosgood committed:
Merge pull request #175 from ryankeleti/ega2-2

fixed error

thosgood committed:
another page

thosgood committed:
finished II.2.4

thosgood committed:
Merge pull request #176 from ryankeleti/ega2-2

finished II.2.4

thosgood committed:
fixed error

thosgood committed:
Merge pull request #177 from ryankeleti/ega2-2

fixed error

thosgood committed:
still alive!

thosgood committed:
Merge pull request #178 from ryankeleti/ega2-2

still alive!

Commit:
Update ega0-10.tex

Commit:
Update CONTRIBUTORS

Commit:
Update ega0-11.tex

Commit:
Update ega0-14.tex

fo?

Commit:
Update ega0-4.tex

Commit:
Update ega0-7.tex

Commit:
Update ega0-8.tex

Commit:
Update ega0-9.tex

thosgood committed:
Merge pull request #179 from agrothendieck/master

Minor typos

Commit:
Update ega1-10.tex

thosgood committed:
Merge pull request #180 from agrothendieck/master

Update ega1-10.tex - minor typo

Commit:
Update ega2-1.tex

Commit:
Update ega2-2.tex

thosgood committed:
some small typos

thosgood committed:
Merge pull request #181 from agrothendieck/master

Minor typos

Commit:
Update ega3-1.tex

Commit:
Update ega3-3.tex

thosgood committed:
Merge pull request #182 from agrothendieck/master

Typos

thosgood committed:
a page or so

thosgood committed:
Merge pull request #183 from ryankeleti/ega2-2

ega2-2

thosgood committed:
two more pages

thosgood committed:
two more pages

thosgood committed:
Merge remote-tracking branch 'origin/ega2-2' into ega2-2

thosgood committed:
Merge pull request #184 from ryankeleti/ega2-2

more II.2

thosgood committed:
another env

thosgood committed:
a chunk of II.2

thosgood committed:
Merge pull request #185 from ryankeleti/ega2-2

finished II.2.5

thosgood committed:
fixed some errors

thosgood committed:
Merge pull request #186 from ryankeleti/ega2-2

fixed some errors

thosgood committed:
fixed error

thosgood committed:
Merge pull request #187 from ryankeleti/ega2-2

fixed error

thosgood committed:
missing sentence

thosgood committed:
\Gamma_* -> \Gamma_\bullet

thosgood committed:
A_* -> A_\bullet

thosgood committed:
a few more pages

thosgood committed:
finished II.2.6

thosgood committed:
Merge pull request #189 from ryankeleti/ega2-2

II.2.6

thosgood committed:
split up EGA 0 toc

thosgood committed:
Merge branch 'master' of https://github.com/ryankeleti/ega

thosgood committed:
starting II.2.7

Commit:
Update ega4-16.tex

Commit:
Merge branch 'ryankeleti:master' into master

Commit:
Update ega2-1.tex

Commit:
Update ega2-2.tex

Commit:
Update ega2-4.tex

thosgood committed:
Merge pull request #190 from agrothendieck/master

Minor typos

thosgood committed:
small typos

thosgood committed:
II.2.7

thosgood committed:
Merge pull request #191 from ryankeleti/ega2-2

II.2.7

thosgood committed:
links

thosgood committed:
too many checkboxes

thosgood committed:
weird markdown behaviour

thosgood committed:
fixed

thosgood committed:
temporary pdf hosting

thosgood committed:
typo

thosgood committed:
a tiny bit

thosgood committed:
another page

thosgood committed:
bit more of 2-2; style consistency

thosgood committed:
ok fixed some errors

thosgood committed:
typo

thosgood committed:
another page

thosgood committed:
another page

thosgood committed:
yet another page

thosgood committed:
fixed some spacing

thosgood committed:
finished II.2.8

thosgood committed:
finished II.2!

thosgood committed:
Merge branch 'master' into ega2-2

thosgood committed:
Merge pull request #193 from ryankeleti/ega2-2

finished II.2

thosgood committed:
Update README.md

thosgood committed:
Update README.md

thosgood committed:
starting

thosgood committed:
starting

thosgood committed:
a page

thosgood committed:
tiny bit

thosgood committed:
another page

thosgood committed:
tiny bit

thosgood committed:
issue #194

thosgood committed:
Merge branch 'master' of https://github.com/ryankeleti/ega

thosgood committed:
KaTeX now supports `\varprojlim` and `\varinjlim`

ryankeleti committed:
PDF links work, updated

thosgood committed:
a half page

thosgood committed:
finished II.3.1

thosgood committed:
Merge pull request #195 from ryankeleti/II.3

finished EGA II, §3.1

thosgood committed:
starting II.3.2

thosgood committed:
tiny correction

thosgood committed:
small typo

https://github.com/ryankeleti/ega/issues/196

ryankeleti committed:
Close sref to fix build error

thosgood committed:
Update README.md

thosgood committed:
tiny progress and a bit of consistency

thosgood committed:
Merge branch 'II.3' of github.com:ryankeleti/ega into II.3

thosgood committed:
a page or so more of II.3.2

thosgood committed:
Merge branch 'master' into II.3

thosgood committed:
finished II.3.2

thosgood committed:
starting II.3.3

thosgood committed:
a few pages of 3.3

thosgood committed:
finished 3.3

thosgood committed:
starting 3.4

thosgood committed:
neighborhood -> neighbourhood for consistency (sorry! can always change back)

thosgood committed:
finished 3.4

thosgood committed:
summary of status

thosgood committed:
finished 3.5

thosgood committed:
fixed errors; compiles

thosgood committed:
finished 3.6

thosgood committed:
starting 3.7

thosgood committed:
another page

thosgood committed:
finished 3.7

thosgood committed:
finished 3.8; finished II.3

thosgood committed:
fixed error

thosgood committed:
(i think the web version is down)

thosgood committed:
Merge pull request #197 from ryankeleti/II.3

EGA II, §3

thosgood committed:
done 2-3!

thosgood committed:
starting 2-6

thosgood committed:
Update README.md

thosgood committed:
Update README.md

thosgood committed:
a few pages

thosgood committed:
another page

thosgood committed:
link to PR

thosgood committed:
finished §6.1

thosgood committed:
single proof

thosgood committed:
finished §6.2

thosgood committed:
another page

thosgood committed:
a bit more

thosgood committed:
finished 6.3

thosgood committed:
starting 6.4

thosgood committed:
tiny bit more

thosgood committed:
half a pag

thosgood committed:
a page

thosgood committed:
finished 6.4

thosgood committed:
starting 6.5

thosgood committed:
a big env

thosgood committed:
an env

Commit:
minor typo

Commit:
Update ega2-2.tex

Commit:
Update ega2-5.tex

Commit:
Update ega2-8.tex

thosgood committed:
Merge pull request #202 from agrothendieck/master

minor typos

thosgood committed:
Merge branch 'master' into 2-6

thosgood committed:
finished 6.5

thosgood committed:
fixed compile errors; starting 6.6

thosgood committed:
a few more pages

thosgood committed:
finished 6.6

thosgood committed:
starting 6.7

thosgood committed:
finished 6.7; finished EGA II!

thosgood committed:
(fixed an arrow label)

thosgood committed:
Merge pull request #198 from ryankeleti/2-6

EGA II, §6

thosgood committed:
Update README.md

thosgood committed:
Update README.md

thosgood committed:
two typos (thanks Ehsan Shahoseini!)

ryankeleti committed:
Delete extra bracket in reference

ryankeleti committed:
Update README.md

Commit:
Update inconsistent labels

thosgood committed:
Merge pull request #204 from wayneyang108/master

Update inconsistent labels

Commit:
Update link between \sref and \label. Correct and add proposio

Commit:
Complete 0.11.3.4 to 0.11.3.6 and add title for 0.11.4 to 0.11.10

Commit:
Finished 0.11.4

thosgood committed:
proofread 0.11.3

thosgood committed:
proofread I.5.2.2

thosgood committed:
proofread up to 0.11.4.5

thosgood committed:
finished proofreading

thosgood committed:
single comma

thosgood committed:
Merge pull request #205 from wayneyang108/master

Finished 0.11.4

calebji0 committed:
Sign up for EGA III.4

thosgood committed:
Merge pull request #207 from calebji0/master

Sign up for EGA III.4

thosgood committed:
Update README.md

thosgood committed:
testing automatic pdf releases

thosgood committed:
trigger build action test

thosgood committed:
lots of latex packages

thosgood committed:
pdf directory

thosgood committed:
woops my mistake

thosgood committed:
build all PDFs

youngsoo-math committed:
Update ega0-3.tex (3.6.2) translation

youngsoo-math committed:
Update ega0-3.tex (3.2.5) translation

youngsoo-math committed:
Update ega0-3.tex

thosgood committed:
Merge pull request #209 from youngsoo-math/master

[ega0-3.tex] translation updated

eliasguisado committed:
Update ega2-2.tex

thosgood committed:
Merge pull request #211 from eliasgv3/master

Transcription typo in ega2-2.tex

eliasguisado committed:
Update ega0-4.tex

eliasguisado committed:
Merge remote-tracking branch 'upstream/master'

thosgood committed:
Merge pull request #212 from eliasgv3/master

Unimportant transcription typo

eliasguisado committed:
Update ega2-3.tex

eliasguisado committed:
Update ega2-3.tex

eliasguisado committed:
Transcription typo

eliasguisado committed:
Update ega2-3.tex

Grothendieck cites I.1.6.4, but upon inspection it is clear he really meant I.1.6.5. I have checked his errata lists in later EGA volumes and this typo is not mentioned.

eliasguisado committed:
Update ega2-2.tex

eliasguisado committed:
Update ega2-2.tex

eliasguisado committed:
Update ega2-2.tex

eliasguisado committed:
Update ega2-5.tex

eliasguisado committed:
Update ega0-7.tex

eliasguisado committed:
Update ega2-2.tex

eliasguisado committed:
Update ega2-3.tex

thosgood committed:
Merge pull request #213 from eliasguisado/master

Typos from already translated material

ryankeleti committed:
Fix I.1.3.14 #214

direct sum => sum
